staker/stakerapp.go

```diff
@@ -2146,7 +2146,7 @@ func (app *StakerApp) ListActiveFinalityProviders(limit uint64, offset uint64) (
 // This function returns control to the caller after step 3. Later is up to the caller
 // to check what is state of unbonding transaction
 func (app *StakerApp) UnbondStaking(
-	stakingTxHash chainhash.Hash, feeRate *btcutil.Amount) (*chainhash.Hash, error) {
+	stakingTxHash chainhash.Hash, _ *btcutil.Amount) (*chainhash.Hash, error) {
```

Comment:
this `feeRate` parameter can be removed completly, this is ancient remnant of post signing un-bonding flow

Reply:
sounds good